client_plugin/utils/plugin_server/plugin_server_windows.go

```diff
@@ -26,7 +27,34 @@ import (
 	"github.com/docker/go-plugins-helpers/volume"
 )
 
-const npipeAddr = `\\.\pipe\vsphere-dvs` // Plugin's npipe address
+const (
+	// npipeAddr is the plugin's npipe address.
+	npipeAddr = "//./pipe/vsphere-dvs"
+
+	// pluginsPath is the Docker plugins directory path.
+	pluginsPath = `C:\ProgramData\docker\plugins\`
+
+	// configFilePath is the plugin's config file path.
+	configFilePath = pluginsPath + "vsphere.json"
+)
+
+// pluginConfig is the plugin config.
+var pluginConfig = fmt.Sprintf(`{ "Name": "vsphere", "Addr": "npipe://%s" }`, npipeAddr)
+
+// writePluginConfig writes the plugin config to the Docker plugins directory.
+func writePluginConfig() {
+	if err := os.MkdirAll(pluginsPath, 0755); err != nil {
+		log.WithFields(log.Fields{"path": pluginsPath,
+			"err": err}).Fatal("Failed to create plugin config directory ")
+		panic("Failed to create plugin config directory")
```

Comment:
Failing with panic doesn't sound a good idea. I'd suggest to return err and fail in a consistent way as we did at line 82.